packages/common/src/store/pages/search-results/actions.ts

```diff
@@ -58,19 +62,24 @@ export type SearchPageActions =
 
 // Query-based search
 
-export const fetchSearchPageResults = (
-  searchText: string,
-  searchKind: SearchKind,
-  limit: number,
+type FetchSearchPageResultsActionArgs = {
+  searchText: string
+  kind: SearchKind
+  limit: number
   offset: number
+  genre?: Genre
+  mood?: Mood
+  isVerified?: boolean
+}
+
+export const fetchSearchPageResults = (
+  args: FetchSearchPageResultsActionArgs
 ): SearchPageActions => ({
   type: FETCH_SEARCH_PAGE_RESULTS,
-  searchText,
-  searchKind,
-  limit,
-  offset
+  ...args
 })
 
+// TODO: update the rest of these actions to object args
```

Comment:
should probably write your name or make a ticket so that this isn't left to the void lol

Reply:
I'll update the actions now!